### PR TITLE
Login

### PR DESCRIPTION
**Pull request by Julianhm9612**
## Summary
- Added new login view
- Added some styles to login
- Added new styles for inputs and buttons
- Added new color variable

## Screenshot
Captura de Pantalla 2021-05-13 a la(s) 3 41 35 p  m

## Trello Card
https://trello.com/c/R2GuW3V9/32-login

### CODE REVIEW

File: books/login.html [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+                <form>
+                    <div class="form-field">
+                        <label class="label" for="email">Email</label>
+                        <input class="input" type="text" id="email" name="email">
+                    </div>
+                    <div class="form-field">
+                        <label class="label" for="password">Password</label>
+                        <input class="input" type="text" id="password" name="password">
+                    </div>
+                    <button class="button primary" type="button">Login</button>
```
Comment (mcallegari10) [inline books/login.html]: type submit

File: books/login.html [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+                    <button class="button secondary" type="button">Sign Up</button>
```
Comment (mcallegari10) [inline books/login.html]: Maybe this should be a link tag 🤔

File: books/scss/styles.scss [hunk 3]
```diff
@@ -51,3 +51,38 @@ body {
   font-size: 20px;
   line-height: 24px;
 }
+
+.form-field {
+  margin: 10px 0;
+}
+
+.label {
+  display: block;
```
Comment (mcallegari10) [inline books/scss/styles.scss]: Is this needed? 🤔

File: books/scss/login.scss [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+    height: auto;
+    width: 212px;
+  }
+
+  .primary {
+    position: relative;
+    margin-bottom: 30px;
+
+    &::after {
+      border-bottom: 2px solid #A8A8A8;
```
Comment (mcallegari10) [inline books/scss/login.scss]: Remember to use variables for colors

**Review by mcallegari10 — CHANGES_REQUESTED** [review]

File: books/login.html (hunk 1 above)
Comment (Julianhm9612) [reply books/login.html]: Have been added

File: books/login.html (hunk 2 above)
Comment (Julianhm9612) [reply books/login.html]: Has been changed :relaxed:

File: books/scss/styles.scss (hunk 3 above)
Comment (Julianhm9612) [reply books/scss/styles.scss]: It has been removed

File: books/scss/login.scss (hunk 4 above)
Comment (Julianhm9612) [reply books/scss/login.scss]: Has been changed

**Review by mcallegari10 — APPROVED** [review]

**Pull request merged**